functions/installEmuGA.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+# Generic Applications
+installEmuGA(){
```

Comment:
i think the name of this should not include Emu.

probably just installGAAppImage or something?